src/pages/home/sidebar/SidebarLinksData.tsx

```diff
@@ -39,8 +39,14 @@ function SidebarLinksData({insets, isLoadingApp = true, onLinkClick, priorityMod
 
     const {orderedReportIDs, currentReportID, policyMemberAccountIDs} = useReportIDs();
 
-    // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
-    useEffect(() => Policy.openWorkspace(activeWorkspaceID ?? '-1', policyMemberAccountIDs), [activeWorkspaceID]);
+    useEffect(() => {
+        if (!activeWorkspaceID) {
+            return;
+        }
+
+        Policy.openWorkspace(activeWorkspaceID, policyMemberAccountIDs);
+        // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
+    }, [activeWorkspaceID]);
```

Comment:
Re-added these since I'm not entirely sure if there are performance implications for not re-rendering every time policyMemberAccountIDs changes. So this just keeps it functionally the same but now we don't call openWorkspace when it's empty